catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -936,4 +951,17 @@ private void applyFilters(OpenSearchFilterVisitorObject visitor, WebClient clien
             client.replaceQueryParam(URL_SRC_PARAMETER, "");
         }
     }
+
+    private List<Metacard> processAdditionalForeignMarkups(Element element, String id) {
+        List<Metacard> metacards = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(markUpSet)) {
+            XMLOutputter xmlOutputter = new XMLOutputter();
+            if (markUpSet.contains(element.getName())) {
```

Comment:
So we are only concerned with matching the local name?  Should we be concerned with namespaces?

Reply:
Right now, matching the local name is sufficient for our needs.  If it is desired to also (or instead) match the namespace, we can make that change.